app/dashboard/_components/InterviewsTable/Columns.tsx

```diff
@@ -146,7 +178,11 @@ export const InterviewColumns = (): ColumnDef<
         return <Badge variant="secondary">Not exported</Badge>;
       }
 
-      return <TimeAgo date={row.original.exportTime} />;
+      return (
+        <div className="text-xs">
+          <TimeAgo date={row.original.exportTime} />
+        </div>
```

Comment:
Since `TimeAgo` renders a `<time>` element, it makes more sense to update it to accept an optional `classNames` property (by making its props extend the element). This will make it more useful as a component and remove the need to wrap it to change styling.

Reply:
resolved [9534b0b](https://github.com/complexdatacollective/Fresco/pull/363/commits/9534b0bcd5869cf3783e688c5ebcb80aaec1f4c7)